flink-runtime-web/web-dashboard/src/app/app.interceptor.ts

```diff
@@ -62,10 +65,8 @@ export class AppInterceptor implements HttpInterceptor {
           ignoreErrorUrlEndsList.every(url => !res.url.endsWith(url)) &&
           ignoreErrorMessage.every(message => errorMessage !== message)
         ) {
-          this.injector.get<StatusService>(StatusService).listOfErrorMessage.push(errorMessage);
-          this.injector
-            .get<NzNotificationService>(NzNotificationService)
-            .info('Server Response Message:', errorMessage.replaceAll(' at ', '\n at '), option);
+          this.statusService.listOfErrorMessage.push(errorMessage);
+          this.notificationService.info('Server Response Message:', errorMessage.replaceAll(' at ', '\n at '), option);
```

Comment:
So was this just completely broken, or...?

(if so, something to backport to 1.17?)

Reply:
the problem was that we were not able to resolve notification service; this just makes the problem more visible, because it needs to be resolved at the construction time

the actual fix is in `main.ts`:

```
    importProvidersFrom(NzNotificationModule),
```

let me try if we have the same issue in 1.17 .. this might have been broken at any point since we don't have regression tests for UI :(